api/model/src/main/java/org/projectnessie/model/EntriesResponse.java

```diff
@@ -87,8 +87,8 @@ static Entry entry(ContentKey name, Content.Type type, String contentId) {
       return ImmutableEntry.of(name, type, contentId, null);
     }
 
-    static Entry entry(ContentKey name, Content.Type type, Content content) {
-      return ImmutableEntry.of(name, type, content.getId(), content);
+    static Entry entry(ContentKey name, Content content) {
```

Comment:
You can remove the redundant cas on line 83 now.

Reply:
undid the change